src/unstract/sdk/audit.py

```diff
@@ -33,23 +24,26 @@ def push_usage_data(
         self,
         platform_api_key: str,
         token_counter: TokenCountingHandler = None,
-        workflow_id: str = "",
-        execution_id: str = "",
-        external_service: str = "",
+        model_name: str = "",
         event_type: CBEventType = None,
+        **kwargs,
     ) -> None:
         """Pushes the usage data to the platform service.
 
         Args:
+            platform_api_key (str): The platform API key.
             token_counter (TokenCountingHandler, optional): The token counter
-              object. Defaults to None.
+                object. Defaults to None.
+            model_name (str, optional): The name of the model.
+                Defaults to "".
+            event_type (CBEventType, optional): The type of the event. Defaults
+                to None.
             workflow_id (str, optional): The ID of the workflow. Defaults to "".
             execution_id (str, optional): The ID of the execution. Defaults
-              to "".
-            external_service (str, optional): The name of the external service.
-              Defaults to "".
-            event_type (CBEventType, optional): The type of the event. Defaults
-              to None.
+                to "".
+            adapter_instance_id (str, optional): The adapter instance ID.
+                Defaults to "".
+            run_id (str, optional): The run ID. Defaults to "".
```

Comment:
NIT: Update kwargs as a separate section and retain only actually args here

Reply:
Done